lib/travis/api/v3/queries/build.rb

```diff
@@ -33,5 +35,12 @@ def restart(user)
         payload
       end
     end
+
+    def priority(user)
+      raise NotFound, "Job not found" if find.jobs.blank?
+      build_job = find.jobs.find_by_commit_id(find.commit_id)
+      priority_status = build_job.update_column(:priority, PRIORITY[:high]) if build_job
+      raise UnprocessableEntity unless priority_status
+    end
```

Comment:
Could you please elaborate what is the intension here, and how do you intend to update the `priority` of all jobs associated with the build, this seems to be updating just one job.

A build can have multiple jobs.

Reply:
Yes, @shairyar previously I was updating single job. I have changed code to set high priority for multiple jobs those are associated with build.  Please review updated code.